gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/oneconvergence_service_node_driver.py

```diff
@@ -678,9 +678,12 @@ def _fetch_template_and_params_for_update(self, context):
                 stitching_port_fip = context.l3_plugin.get_floatingip(
                     admin_context,
                     floatingips[0]['id'])['floating_ip_address']
-
+                remote_id = context.core_plugin.get_port(
+                    admin_context,
+                    stitching_port_id)['fixed_ips'][0]['ip_address']
                 desc = ('fip=' + floating_ip + ";tunnel_local_cidr=" +
-                        provider_subnet['cidr']+";user_access_ip=" + stitching_port_fip)
+                        provider_subnet['cidr'] + ";user_access_ip=" +
+                        stitching_port_fip + ";remote_id=" + remote_id)
```

Comment:
It would be better to re-name this as fixed_ip. It is remote_id for the peer, not for the VPNaaS